_data/companies/optimize.yml

```
@@ -0,0 +1,6 @@
+name: Optimize Enterprises, Public Benefit Corporation (www.optimize.me) 
```

```suggestion
name: Optimize Enterprises, Public Benefit Corporation
```
We try to avoid linking directly to the company's home page unless the announcement is front and center.